src/main/java/io/orkes/conductor/client/ApiClient.java

```diff
@@ -112,9 +125,9 @@ public ApiClient(
         this.ssmKeyPath = keyPath;
         this.ssmSecretPath = secretPath;
         try {
-            this.refreshToken();
-        } catch (Exception e) {
-            LOGGER.warn("Failed to set authentication token. Reason: " + e.getMessage());
+            getToken();
+        } catch (Throwable t) {
+            LOGGER.error(t.getMessage(), t);
```

Comment:
Better to have a meaningful message and throwable instance as the second argument.

Reply:
Please check all the error message in this PR.